backend/grant/app.py

```diff
@@ -48,6 +48,10 @@ def handle_error(err):
             return jsonify({"message": error_message}), err.code, headers
         else:
             return jsonify({"message": error_message}), err.code
+    
+    @app.errorhandler(Exception)
+    def handle_exception(err):
```

Comment:
Should we have some logging here?

Reply:
For sure, and Sentry logging too.

Reply:
@wbobeirne Should I keep this open until logging is added?

Reply:
I'll add the logging with the Sentry rewrite, since I'll need to change the way we do it anyway.

Reply:
Feel free to merge 👌